python/utils/generatejson.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" This script generates request json for the Vision API.
+You can run this script like this:
+python generatejson.py \
+    -i <inputfile> \
+    -o <outputfile.json>
```

Comment:
Suggestion: I kind of like in markdown where you indent code blocks - makes it easier to parse